projects/lodepng/Dockerfile

```diff
@@ -19,4 +19,5 @@ MAINTAINER lvandeve@gmail.com
 RUN apt-get update && apt-get install -y make autoconf automake libtool
 RUN git clone --depth 1 https://github.com/lvandeve/lodepng.git lodepng     # or use other version control
 WORKDIR lodepng
-COPY build.sh $SRC/
+ADD http://oss-fuzz-corpus.storage.googleapis.com/lodepng/lodepng_fuzzer_seed_corpus.zip $SRC/
```

Comment:
Where did you get this seed corpus from, can we add the direct location here, instead of using gs://oss-fuzz-corpus.storage

Reply:
The seed corpus was gathered from these two places:
1. Seed PNGs Link 1: [https://lcamtuf.coredump.cx/afl/demo/](https://lcamtuf.coredump.cx/afl/demo/)
2. Seed PNGs Link 2: [https://github.com/glennrp/libpng/tree/libpng16/contrib/pngsuite](https://github.com/glennrp/libpng/tree/libpng16/contrib/pngsuite)
Combined they have more than 2000 files, so I have taken a subset of them and merged them using libFuzzer and made final 105 files. So, that's why @oliverchang asked me to upload it on bucket and use it from there.

Reply:
That is fine then.